src/pages/Examples/TimeRangePicker/TimeRangeList.js

```diff
@@ -0,0 +1,42 @@
+import { v4 as uuidv4 } from "uuid";
+
+export const TimeRangeList = ({selected = false}) => {
+
+  const rangeValues = [
```

Comment:
If the array is in the component, then you must either move the array out of the component, or use `useMemo`